src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -70,6 +70,7 @@ protected function configure(): void
                 new InputOption('time-limit', 't', InputOption::VALUE_REQUIRED, 'The time limit in seconds the worker can handle new messages'),
                 new InputOption('sleep', null, InputOption::VALUE_REQUIRED, 'Seconds to sleep before asking for new messages after no messages were found', 1),
                 new InputOption('bus', 'b', InputOption::VALUE_REQUIRED, 'Name of the bus to which received messages should be dispatched (if not passed, bus is determined automatically)'),
+                new InputOption('queues', null, InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'To limit consuming to a only some queues (for receivers that support it)'),
```

Comment:
The sentence sounds a bit weird 🙂

Reply:
hm, indeed. i tried to rephrase it. better?

Reply:
Much better, thank you!